js/ui/windowManager.js

```diff
@@ -1934,6 +1934,7 @@ const CodingManager = new Lang.Class({
         this._rotateInActors = [];
         this._rotateOutActors = [];
         this._firstFrameConnections = [];
+        this._previousFocusedWindow = null;
```

Comment:
Yeah, I had a look as well and could not find anything that would fit these needs.